R/utilities.R

```diff
@@ -1292,14 +1292,30 @@ test_glue = function(placeholder="INSERTED"){
 #' @param case_set Optional short name for a pre-defined set of cases.
 #' @param sbs_manipulation Optional variable for transforming sbs values (e.g log, scale).
 #' @param seq_type_filter Filtering criteria, default is genomes.
-#' @param from_cache Boolean variable for using cached results, default is TRUE.
+#' @param from_cache Boolean variable for using cached results, default is TRUE. If write_to_file is TRUE, this aprameter auto-defaults to FALSE. 
 #'
 #' @return A table keyed on biopsy_id that contains a bunch of per-sample results from GAMBL
```

Comment:
For the sake of our future sanity can you add some details in this section that tell the user where the cached results are written?